core/src/main/java/com/yahoo/ycsb/generator/CounterGenerator.java

```diff
@@ -32,23 +32,18 @@
 	public CounterGenerator(int countstart)
 	{
 		counter=new AtomicInteger(countstart);
-		setLastInt(counter.get()-1);
```

Comment:
right but setLastValue now exists in NumberGenerator and isn't called? If you call getLastString it calls lastValue() which won't be populated by this CounterGenerator since setLastValue is what sets the lastValue?